cluster/service/cluster.go

```diff
@@ -38,25 +32,25 @@ func NewClusterService(context servicectx.ServiceContext, loader ConfigLoader) s
 	}
 }
 
-// CreateOrSaveOSOClusterFromConfig creates clusters or save updated cluster info from config
-func (c clusterService) CreateOrSaveOSOClusterFromConfig(ctx context.Context) error {
-	for _, osoCluster := range c.loader.GetOSOClusters() {
+// CreateOrSaveClusterFromConfig creates clusters or save updated cluster info from config
+func (c clusterService) CreateOrSaveClusterFromConfig(ctx context.Context) error {
+	for _, configCluster := range c.loader.GetClusters() {
 		cluster := &repository.Cluster{
-			Name:              osoCluster.Name,
-			URL:               httpsupport.AddTrailingSlashToURL(osoCluster.APIURL),
-			ConsoleURL:        httpsupport.AddTrailingSlashToURL(osoCluster.ConsoleURL),
-			MetricsURL:        httpsupport.AddTrailingSlashToURL(osoCluster.MetricsURL),
-			LoggingURL:        httpsupport.AddTrailingSlashToURL(osoCluster.LoggingURL),
-			AppDNS:            osoCluster.AppDNS,
-			CapacityExhausted: osoCluster.CapacityExhausted,
+			Name:              configCluster.Name,
+			URL:               httpsupport.AddTrailingSlashToURL(configCluster.APIURL),
+			ConsoleURL:        httpsupport.AddTrailingSlashToURL(configCluster.ConsoleURL),
+			MetricsURL:        httpsupport.AddTrailingSlashToURL(configCluster.MetricsURL),
+			LoggingURL:        httpsupport.AddTrailingSlashToURL(configCluster.LoggingURL),
+			AppDNS:            configCluster.AppDNS,
+			CapacityExhausted: configCluster.CapacityExhausted,
+			Type:              configCluster.Type,
```

Comment:
what if the type is missing from the config?

Reply:
it won't be because we are adding `type` to `OSO`. Look at https://github.com/fabric8-services/fabric8-cluster/pull/32/files#diff-786c8376feaa68e9ebee88ca950d2774R208

Reply:
ok, thanks!